fastai/data/external.py

```diff
@@ -110,15 +112,24 @@ class URLs():
     WT103_FWD          = f'{S3_MODEL}wt103-fwd.tgz'
     WT103_BWD          = f'{S3_MODEL}wt103-bwd.tgz'
 
-    def path(url='.', c_key='archive'):
+    def path(
+        url='.', # URL of file to download
+        c_key='archive' # Name of a folder in `fastai_cfg` to save the file. Should be one of [`'data', 'archive', 'storage', 'model'`]
+    ) -> Path: # Path to which the file will be downloaded
         "Return local path where to download based on `c_key`"
         fname = url.split('/')[-1]
         local_path = URLs.LOCAL_PATH/('models' if c_key=='models' else 'data')/fname
         if local_path.exists(): return local_path
         return fastai_path(c_key)/fname
 
 # Cell
-def untar_data(url, archive=None, data=None, c_key='data', force_download=False):#, extract_func=file_extract, timeout=4):
+def untar_data(
+    url, # URL of file to download
+    archive=None, # Value for `'archive'` in `fastai_cfg`. If None, default value from `config.ini` is used
+    data=None, # Value for `'data'` in `fastai_cfg`. If None, default value from `config.ini` is used
```

Comment:
Same nit as earlier, `None` is a type:
```suggestion
    data=None, # Value for `'data'` in `fastai_cfg`. If `None`, default value from `config.ini` is used
```